### PR TITLE
e2e shared-context

### PR DESCRIPTION
**Pull request by Andranik-mikhaielian**
Screenshot 2022-12-20 at 3 24 46 PM
Screenshot 2022-12-20 at 3 25 07 PM

### CODE REVIEW

File: shared-context/e2e/checkApp1.cy.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        basePage.checkElementWithTextPresence({
+            selector: baseSelectors.h2,
+            text: Constants.elementsText.sharedContextApp1H2
+        })
+        basePage.checkElementWithTextPresence({
+            selector:baseSelectors.paragraph,
+            text: Constants.elementsText.sharedContextApp1Paragraph
+        })
+    })
+})
```
Comment (PeaceDate) [inline shared-context/e2e/checkApp1.cy.ts:25]: Add blank line in the end

File: shared-context/e2e/checkApp2.cy.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        basePage.checkElementWithTextPresence({
+            selector: baseSelectors.h2,
+            text: Constants.elementsText.sharedContextApp2H2
+        })
+        basePage.checkElementWithTextPresence({
+            selector:baseSelectors.paragraph,
+            text: Constants.elementsText.sharedContextApp2Paragraph
+        })
+    })
+})
```
Comment (PeaceDate) [inline shared-context/e2e/checkApp2.cy.ts:25]: Add blank line in the end

File: shared-context/package.json [hunk 3]
```diff
@@ -7,6 +7,7 @@
     "clean": "lerna run --scope @shared-context/* --parallel clean"
   },
   "devDependencies": {
-    "lerna": "3.22.1"
+    "lerna": "3.22.1",
+    "webpack-cli": "^5.0.1"
```
Comment (PeaceDate) [inline shared-context/package.json:11]: Change webpack-cli version to 4.10.0 or delete package.json file from changed file at all

**Comment by ScriptedAlchemy** [comment]
some git conflicts but can merge once duxed

**Pull request merged**